packages/combobox/src/Combobox/Combobox.types.ts

```diff
@@ -109,9 +109,16 @@ export type BaseComboboxProps = Omit<HTMLElementProps<'div'>, 'onChange'> &
 
     /**
      * The message shown below the input when state is `error`
+     * @default 'This input needs your attention'
```

Comment:
feel like we should have these as consts external to this package if we're going to leverage them in other places